pyaedt/hfss.py

```diff
@@ -5417,8 +5418,11 @@ def get_antenna_ffd_solution_data(
         sphere=None,
         variations=None,
         overwrite=True,
+        link_to_hfss=True,
     ):
-        """Export antennas parameters to Far Field Data (FFD) files and return the ``FfdSolutionDataExporter`` object.
+        """Export antennas parameters to Far Field Data (FFD) files and return an
```

Comment:
```suggestion
        """Export the antenna's parameters to Far Field Data (FFD) files and return an
```

Reply:
@PipKat In this context "antenna parameter" is correct or the plural. This is clear to engineers working with antennas.